docs/dataset-overviews/anthropogenic-slcf-co2-emissions.md

```diff
@@ -10,18 +10,19 @@
 
```

Comment:
@rhoesly @ssmithClimate do you want to make any comments/additions to these updates?

Reply:
I'll merge now to keep things moving. Open a new MR with any follow up changes if they're needed, thanks